CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - Updated layers:
   - "Earthquakes" updated to include 2021 data
+  - "Monthly Mean Concentration (25km)" updated to include 2021 data
```

Comment:
Lets note that we are specifically updating the "Feb or March (max monthly extent)" group to include 2021 data.